docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,241 @@
+
+1. On the locations page of the [Viam app](https://app.viam.com), add a new robot by providing a name in the **New Robot** field and clicking **Add robot**.
+2. Click on that robot's name to go to its page.
+
+## Configure a camera
+
+Navigate to the **Config** tab of your machine's page on the [Viam app](https://app.viam.com).
+Configure the camera you want to use for your security system.
+
+We configured ours as a webcam named `"my-webcam"`.
+Follow [these instructions](/components/camera/webcam/) to do so.
+
+Connect your camera to your machine running `viam-server`, and hang it up somewhere where it can see people in your home:
+
+{{<imgproc src="/tutorials/verification-system/camera.jpeg" resize="500x" declaredimensions=true alt="Camera hanging up in office.">}}
+
+Next, configure the person detector, or, the "coarser" layer of the security system that verifies that there's a person moving.
+
+## Configure an `mlmodel` person detector
+
+Create a detector to detect when people are in your verification system camera's field of vision:
+
+1. [Capture images from your camera](/data/capture/#configure-data-capture-for-individual-components) with Viam's data management service, then [create a dataset](/data/dataset/#create-a-dataset-and-add-data).
+2. Label your data with [bounding boxes](/data/dataset/#bounding-boxes) labeled as `Person` where a person is present to create an object detection model for people. You only want this model to be able to distinguish between what is and isn't a person, so you can conduct this training step with anyone.
+3. [Train a model on your dataset](/ml/train-model/). Select **Model Type** as **Object Detection**.
+4. [Deploy the model](/ml/deploy/) to your machine so you can use it.
+
+Now that you've trained and deployed the model itself, [configure an `mlmodel` detector model](/ml/vision/detection/#configure-an-mlmodel-detector) with the name of the ML model service you deployed your model as.
+Name this detector `"people-detect"`.
+
+For example, we configured a person detector as follows:
+
+```json {class="line-numbers linkable-line-numbers"}
+"services": [
+  {
+    "type": "mlmodel",
+    "model": "tflite_cpu",
+    "attributes": {
+      "model_path": "${packages.persondetect0tflite}/persondetect0.tflite",
+      "label_path": "${packages.persondetect0tflite}/persondetectlabels.txt",
+      "num_threads": 1
+    },
+    "name": "persondetect"
+  },
+  {
+    "name": "people-detect",
+    "type": "vision",
+    "model": "mlmodel",
+    "attributes": {
+      "mlmodel_name": "persondetect"
+    }
+  }
+]
+```
+
+This will serve as the first and second layers of your verification system.
+Now, move on to configuring the "finer" layer of the detectors-- the facial recognition detector.
+
+## Configure a facial detector
+
+Now, to disarm our alarm, we want our verification system to be able to give someone an "ok" based on their ID.
```

Comment:
So for this to make sense you need to explain that the module that we will be using uses the first detector to start a countdown to an alarm. You did in the introduction but that's a long time ago. It may also make sense to explain that we will add a module that contains the code that runs the alarm system based on the given detectors before we dive into creating the detectors.
A diagram with the module and the component and services would be very helpful

Reply:
done / made ticket